client/signup/tailored-flow-processing-screen/index.jsx

```diff
@@ -91,9 +91,11 @@ export default function TailoredFlowProcessingScreen( { flowName } ) {
 				/>
 			</div>
 
-			<div className="reskinned-processing-screen__jetpack-powered">
-				<JetpackLogo monochrome size={ 18 } /> <span>Jetpack powered</span>
-			</div>
+			{ flowName === 'newsletter' && (
```

Comment:
Should use the [constant](https://github.com/Automattic/wp-calypso/blob/trunk/packages/onboarding/src/utils/flows.ts#L1) from onboarding.

Reply:
Absolutely, thank you, the next commit will address it.